src/platform_impl/windows/event_loop.rs

```diff
@@ -853,7 +853,9 @@ unsafe extern "system" fn public_window_callback<T: 'static>(
                     winuser::MonitorFromRect(&new_rect, winuser::MONITOR_DEFAULTTONULL);
                 match fullscreen {
                     Fullscreen::Borderless(ref mut fullscreen_monitor) => {
-                        if new_monitor != fullscreen_monitor.inner.hmonitor()
+                        // Unwrap is safe, since we explicitly map `None` to current monitor
+                        // when we're setting fullscreen state.
+                        if new_monitor != fullscreen_monitor.as_ref().unwrap().inner.hmonitor()
```

Comment:
This just seems ripe for failure. I don't like the mapping things and then making assumptions later structure.

Reply:
Well, you can't store it with `None`, so you always store it unwraped, one more thing that you can't access monitor here.

Reply:
I'll ask for current monitor, since it should be available here.

Reply:
Oh, nvm, it's acually important to store those things unwrapped and not in Option for fullscreen. So how we should approach that? I'm fine with `unwrap` here tbqh.